Packs/CommonScripts/Lists/PrivateIPs.json

```diff
@@ -0,0 +1,3 @@
+10.0.0.0/8, IANA_Private_Address
+172.16.0.0/12, IANA_Private_Address
+192.168.0.0/16, IANA_Private_Address
```

Comment:
invalid json, FYI

Reply:
True, Anar suggested changing the file extension because the SDK fails to upload non-JSON lists.
The contents of the list should not be in JSON structure